task/kubernetes-actions/0.1/kubernetes-actions.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kubectl-actions
```

Comment:
Can we have file name and directory name as `kubectl-actions` only please to match the filename and directory name similar to resource name